src/_poplink/prototyping-deck.md

```diff
@@ -0,0 +1,7 @@
+[Buy in store](https://store.bitcraze.io/products/prototyping-deck)
+
+[Product Page](/products/prototyping-deck)
+
+[Other deck compatibility](/documentation/system/platform/cf2-expansiondecks/#deck---deck)
```

Comment:
Actually this can be removed, as it's not in the table (ofcourse this is related on how the user prototypes their deck). Perhaps refer them to the Deck pin allocation instead?